src/domain/application/applicationView/ApplicationView.test.tsx

```diff
@@ -224,6 +224,32 @@ describe('Excavation announcement application view', () => {
     expect(screen.getByText('Lataa työ valmis (PDF)')).toBeInTheDocument();
   });
 
+  test('Shows correct information in areas tab', async () => {
+    const { user } = render(<ApplicationViewContainer id={5} />);
+    await waitForLoadingToFinish();
+    await user.click(screen.getByRole('tab', { name: /alueet/i }));
+    const { queryByText } = within(screen.getByRole('tabpanel', { name: /alueet/i }));
+
+    expect(queryByText('12.1.2023')).toBeInTheDocument();
+    expect(queryByText('12.11.2024')).toBeInTheDocument();
+    expect(queryByText('Aidasmäentie 5')).toBeInTheDocument();
+    expect(queryByText('Vesi, Viemäri')).toBeInTheDocument();
+    expect(queryByText('Työalue 1')).toBeInTheDocument();
+    expect(queryByText('Pinta-ala: 158 m²')).toBeInTheDocument();
+    expect(queryByText('Työalue 2')).toBeInTheDocument();
+    expect(queryByText('Pinta-ala: 30 m²')).toBeInTheDocument();
+    expect(queryByText('188 m²')).toBeInTheDocument();
+    expect(screen.getByTestId('test-pyoraliikenneindeksi')).toHaveTextContent('3');
+    expect(screen.getByTestId('test-autoliikenneindeksi')).toHaveTextContent('3');
+    expect(screen.getByTestId('test-linjaautoliikenneindeksi')).toHaveTextContent('4');
+    expect(screen.getByTestId('test-raitioliikenneindeksi')).toHaveTextContent('5');
+    expect(queryByText('3: Toistuva meluhaitta')).toBeInTheDocument();
```

Comment:
According to Figma, these nuisances do not have their numeric values displayed. E.g. "Toistuva meluhaitta" instead of "3: Toistuva meluhaitta".

Reply:
Good point. Removed numeric values.